readthedocs/config/config.py

```diff
@@ -370,6 +382,29 @@ def validate_build_config_with_os(self):
         build["apt_packages"] = self.validate_apt_packages()
         return build
 
+    def validate_build_jobs_build(self, build_jobs):
+        # The build.jobs.build key is optional.
+        if "build" not in build_jobs:
+            return None
+
+        result = {}
+        build_jobs_build = build_jobs["build"]
+        validate_dict(build_jobs_build)
+
+        if not "html" in build_jobs_build:
+            raise ConfigError(message_id=ConfigError.HTML_BUILD_STEP_REQUIRED)
```

Comment:
I think `build.jobs.build.html` shouldn't be mandatory. That means that users cannot override _only_ any of the other formats like `build.jobs.build.pdf` or `build.jobs.build.epub`.